### ISSUE TITLE
 a few more duplication/display/delete issues

### ISSUE THREAD

**Issue by jlbruce** [USGS-WiM/SiGLDMS#234]
First - I accidentally added the same Org/Div twice (selected from the drop downs), so it showed up twice. When I deleted the second one, it deleted both. It would be nice for it to check and see if the selection (Org, Org/Div, or Org/Div/Sec) is a duplicate and, if it is, not add it to the display list.

Second (related) - To further test this, I added a duplicate Org/Div/Sec that had some projects, and when I deleted the empty one (with no projects), I'm assuming it also deleted the one that HAD projects, which confused it, and I got the same error as before.

**Comment by jlbruce** [comment]
It sounds like maybe some of this will be prevented by your new code - maybe I just caught this in-between updates. Let me know (after you've fixed my deletion issue) and I'll retest.

**Comment by troddick** [comment]
@jlbruce  I agree. I was working on fixing this at the exact time you were working on breaking it. :)
Then endpoint to get the orgResources is working now so the page loads. I had to remove the div_id from the org_system for your test org 'Some new Org1', since I don't know what it was supposed to be. 
Give it a try now.

**Comment by jlbruce** [comment]
Everything is working great now - I wasn't able to break it even a little!
